invenio_records_resources/services/records/results.py

```diff
@@ -70,6 +70,32 @@ def to_dict(self):
             res['errors'] = self._errors
         return res
 
+    def permissions_to(self, actions):
```

Comment:
I would probably make the name a bit more explicit to something like `has_permission_to(...)`

Reply:
Sure. I will go for `has_permissions_to(...)` (keeping the plural because multiple permissions).